### ISSUE TITLE
Feature Request. Multiple terminals and multiple users from a single processor. 

### ISSUE THREAD

**Issue by MFernflower** [MightyPirates/OpenComputers#34]
A mainframe for a university or giant corporation would be nice!

**Comment by ghost** [comment]
#17 ?

**Comment by LordFokas** [comment]
This has been discussed. It would be really hard to control.
You'd need multithreading or at least multitasking for the multiple user stuff to run in parallel, and it would be really hard to control.

I doubt Sangar will do it, and if he does, it won't be soon.

**Comment by MFernflower** [comment]
I see. also there was another issue like this one. Sorry
